src/specs/models/samples-bodies/BeginSampleFileUpload.v2.yaml

```diff
@@ -0,0 +1,21 @@
+title: Begin sample file multipart upload
+description: 'Data to begin an s3 multipart upload'
+type: object
+properties:
+  size:
+    type: number
+  metadata:
+    description: Metadata to append to the s3 object on upload
```

Comment:
the metadata is only cellranger version? what about the sample technologies that aren't 10x?

Reply:
as far as I know 10x this is the only case where we use this s3 metadata thing.

I copied it over from the sample create validation where it was before without changing it so this is the same tha thas been around for over a year

Reply:
ok